UPDATING.md

```diff
@@ -50,6 +50,14 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Rename policy to task_policy
+
+Because Airflow introduced DAG level policy (`dag_policy`) we decided to rename existing `policy`
+function to `task_policy` to make the distinction more profound and avoid any confusion.
+
+Users using cluster policy need to rename their `policy` functions in `airflow_local_settings.py`
+to `task_policy`.
```

Comment:
`policy` would still work but raise deprecation warning and will be completely removed in Airflow 2.1

Reply:
I think this information creates no confusion - users have to take an action (now or latter). The sooner the better in my opinion. Imho we should encourage users to adjust. However, if you thing this may create some doubts then I can change the entry. But it will have to be adjusted once we remove the deprecation

Reply:
hmm... 🤔 yeah maybe keep it as it is